src/services/MemberService.js

```diff
@@ -194,7 +194,7 @@ async function getProfileCompleteness (currentUser, handle, query) {
       data.bio = true
     }
     // TODO: Do we use the short bio or the "description" field of the member object?
-    if(item.traitId=="basic_info" && item.traits.data[0].gigAvailability) {
```

Comment:
Please move to personalization traits. There is the location where profile app will store that preference.

Reply:
Updated, please check.